### DISCUSSION TITLE
https://widcard.win/posts/blog/baipiao-cdn

### DISCUSSION THREAD

**Discussion by giscus** [widcardw/giscus-discussion#10] [Announcements]
# https://widcard.win/posts/blog/baipiao-cdn

Opinionated Vite Starter Template

https://widcard.win/posts/blog/baipiao-cdn

**Comment by widcardw** [comment]
另外，添加 DNS 规则后，仓库中会多一个 CNAME 文件，这个需要 merge 一下
